freqtrade/freqtradebot.py

```diff
@@ -480,6 +480,14 @@ def execute_buy(self, pair: str, stake_amount: float, price: Optional[float] = N
         else:
             # Calculate price
             buy_limit_requested = self.exchange.get_rate(pair, refresh=True, side="buy")
+            if self.config.get('use_custom_entry_price', False):
+                buy_rate = self.exchange.get_rate(pair, refresh=True, side="buy")
+                custom_entry_price = strategy_safe_wrapper(self.strategy.custom_entry_price,
+                                                    default_retval=stake_amount)(
+                    pair=pair, current_time=datetime.now(timezone.utc),
+                    current_rate=buy_rate)
```

Comment:
You'll want to rename the kwarg to `proposed_rate` - to align it to the interface ;)